src/pages/works.js

```diff
@@ -24,15 +23,24 @@ export default function Works({ data }) {
   const { allDesignersYaml } = data
 
   const designers = allDesignersYaml.edges.map(edge => edge.node)
+  const works = designers.reduce((items, d) => items.concat(d.works), [])
```

Comment:
Add sanity check for `d.works`: `items.concat(d.works || [])`

Unless i am mistaken and concat works with `null` :)